libbeat/processors/add_nomad_metadata/nomad.go

```diff
@@ -0,0 +1,164 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package add_nomad_metadata
+
+import (
+	"time"
+
+	nomad "github.com/hashicorp/nomad/api"
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/common"
+	nomadbeat "github.com/elastic/beats/libbeat/common/nomad"
```

Comment:
Could you please rename this alias? nomadbeat sounds like a beat. It could be called `libnomad` or `nomadlib`.

Reply:
Changed to nomadlib